packages/core/src/rules/spot/arazzo-version-enum.ts

```diff
@@ -0,0 +1,19 @@
+import { ARAZZO_VERSIONS_SUPPORTED_BY_SPOT } from '../../typings/arazzo';
+
+import type { ArazzoRule } from '../../visitors';
+import type { UserContext } from '../../walk';
+
+export const ArazzoVersionEnum: ArazzoRule = () => {
+  return {
+    Root: {
+      enter(root, { report, location }: UserContext) {
+        if (!ARAZZO_VERSIONS_SUPPORTED_BY_SPOT.includes(root.arazzo)) {
+          report({
+            message: 'Only `1.0.0` Arazzo version is supported by Spot.',
```

Comment:
When `ARAZZO_VERSIONS_SUPPORTED_BY_SPOT` gets changed, the message will be outdated. Could you print the list of the versions from `ARAZZO_VERSIONS_SUPPORTED_BY_SPOT` in the message instead of hardcoding?